main.go

```diff
@@ -74,6 +79,7 @@ func main() {
 	flag.IntVar(&concurrent, "concurrent", 4, "The number of concurrent notification reconciles.")
 	flag.BoolVar(&watchAllNamespaces, "watch-all-namespaces", true,
 		"Watch for custom resources in all namespaces, if set to false it will only watch the runtime namespace.")
+	flag.IntVar(&rateLimitInterval, "rate-limit-interval", 5, "Interval in which rate limit has effect, value is given in minutes.")
```

Comment:
Let's make this a DurationVar with `5m` as default like here: https://github.com/fluxcd/kustomize-controller/blob/main/main.go#L79